tls/s2n_x509_validator.c

```diff
@@ -874,7 +874,9 @@ S2N_RESULT s2n_x509_validator_validate_cert_stapled_ocsp_response(struct s2n_x50
     int status = 0;
     int reason = 0;
 
-    /* sha1 is the only supported OCSP digest */
+    /* SHA-1 is the only supported hash algorithm for the CertID due to its wide compatibility and 
+     * established use in OCSP responders. 
+     */
```

Comment:
nit: Reading this again, it seems like wide compatibility and established use are kind of saying the same thing?
```suggestion
    /* SHA-1 is the only supported hash algorithm for the CertID due to its established use in 
     * OCSP responders. 
     */
```